less/about/history.less

```diff
@@ -0,0 +1,47 @@
+@import "./itemList.less";
+
+.historyPage {
+  margin: 20px;
+
+  .historyPageContent {
+    border-top: 1px solid @chromeBorderColor;
+    display: flex;
+
+    .historyList {
+      padding-top: 10px;
+      overflow: hidden;
+
+      .listItem {
+        display: flex;
+        height: 1rem;
+
+        .aboutListItem {
+          align-items: center;
+        }
+
+        .aboutItemDate {
+          color: #aaa;
+          margin-right: 10px;
+        }
+      }
+    }
+
+    .sticky-outer-wrapper {
+      min-width: 100%;
+      padding-top: 10px;
+    }
+  }
+}
+
+.searchInput {
+    float: right;
+    padding: 5px;
+    margin-top: -35px;
+}
```

Comment:
there might be an opportunity to refactor some common styles with bookmarks page, if so please take the time to do that now before we merge this.